lib/board.rb

```diff
@@ -11,7 +11,7 @@ def empty?
     end
 
     def full?
-        moves.length == 9
+        moves.length == 9 && !moves.include?(nil)
```

Comment:
Where's the test for this new logic?

Reply:
The test that I already have for the full? method helped me with this new logic.

Reply:
If your tests already cover this logic, then this logic isn't needed. Right?